tests/runtest.cmd

```diff
@@ -533,7 +533,7 @@ goto :eof
         "%CoreRT_CliDir%\dotnet.exe" msbuild /m /ConsoleLoggerParameters:ForceNoAlign "/p:IlcPath=%CoreRT_ToolchainDir%" "/p:Configuration=%CoreRT_BuildType%" "/p:OSGroup=%CoreRT_BuildOS%" "/p:Platform=%CoreRT_BuildArch%" "/p:RepoLocalBuild=true" "/p:FrameworkLibPath=%CoreRT_TestRoot%..\bin\%CoreRT_BuildOS%.%CoreRT_BuildArch%.%CoreRT_BuildType%\lib" "/p:FrameworkObjPath=%~dp0..\bin\obj\%CoreRT_BuildOS%.%CoreRT_BuildArch%.%CoreRT_BuildType%\Framework" /t:CreateLib %CoreRT_TestRoot%\..\src\BuildIntegration\BuildFrameworkNativeObjects.proj
     )
 
-    if "%CoreRT_TestCompileMode%" == "readytorun" (
+    if /i "%CoreRT_TestCompileMode%" == "readytorun" (
```

Comment:
The 'if' statement on the line 528 has same issue. Could you please fix it too?

Reply:
I made all comparisons between a batch variable and non-empty text case-insensitive for consistency